play/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Are we using `npm` for this new one? If yes, probably makes sense to switch to yarn

Reply:
I can switch over to `yarn` for this app. I think that means we'll need to switch over the `yarn` for `central` and `host` as well though. I know we were planning to do that but I'll have to make a point of it prior to any coops joining so don't have confusion. This is more of a self reminder for me, as I'm going to continue focusing on play for now.

Reply:
I have this work done for mobile, networking, and debug panel. Usually the steps are:

* Remove `package-lock.json`
* Run `yarn install`
* Merge the new changes including `yarn.lock`

For Host app, it'd need the changes to `package.json` script section. You can check [this section](https://github.com/rightoneducation/righton-app/pull/519/files#diff-22d39eaa038d1dbabef4c120dd46c830896fdada76c56b7a7757c1ed5e5bb1c5R28-R29) in my PR.

Reply:
Awesome, I'll check that out. Thanks Mani